packages/lesswrong/components/common/HeadTags.jsx

```diff
@@ -19,10 +20,10 @@ class HeadTags extends PureComponent {
     const appTitle = getSetting('forumSettings.tabTitle', 'LessWrong 2.0');
     const title = this.props.title ? `${this.props.title} - ${appTitle}` : appTitle;
     const description = this.props.description || getSetting('tagline') || getSetting('description');
-
+    
     return (
       <div>
-        <Helmet>
+        <Helmet key={this.props.location.href}>
```

Comment:
I was pretty sure the API here should be `location.pathName` or something like that, at least according to: 

https://reacttraining.com/react-router/core/api/location